### ISSUE TITLE
[Question] How to use launchPersistentContext properly to open a non-incognito tab?

### ISSUE THREAD

**Issue by iamandras** [microsoft/playwright#13878]
Hi,

I would like to use playwright to automate some stuff in my usual Chrome browser.

Based on the documentation I should browserType.launchPersistentContext to open a tab that have access to my Chrome's local storage and cookies.

I wrote chrome://version to my normal Chrome to get my profile path.

The script below opens gmail in Chrome, but I'm not logged in.

What do I do wrong? 

Thanks for your help.
Andras

```
const { chromium } = require('playwright');

function wait(ms) {
    return new Promise((resolve, reject) => {
        setTimeout(() => {resolve()}, ms);
    });
}

async function test() {
    const context = await chromium.launchPersistentContext(
      '/Users/andras/Library/Application Support/Google/Chrome/Default',
    {
        headless: false,
        channel: 'chrome',
    });

    const page = await context.newPage();
    await page.goto('https://gmail.com');

    await wait(15000);

    await page.close();
    await context.close();
}

test();
```

**Comment by iamandras** [comment]
I just tried a modified script on Windows: I was able to open a gmail that is logged in.
But I had to fully close Chrome before I could do it.
Same approach didn't work on MacOS.

```
        const context = await chromium.launchPersistentContext(
            'C:\\Users\\szabo\\AppData\\Local\\Google\\Chrome\\User Data',
            { headless: false, channel: 'chrome' });

```

**Comment by dgozman** [comment]
[Docs](https://playwright.dev/docs/api/class-browsertype#browser-type-launch-persistent-context) say that Chromium's user data directory is the **parent** directory of the "Profile Path" seen at `chrome://version`. In your case, that would be `/Users/andras/Library/Application Support/Google/Chrome`.

**Comment by iamandras** [comment]
Thx for the answer.
Yes, I've checked the docs and the userDataDir path works on windows. But not on a Mac.

Also, I just realized that it might not be possible what I would like to achieve.
Playwright always needs to open a new Chrome, it can't connect to an existing, running instance.
Is it true?

**Comment by dgozman** [comment]
> Playwright always needs to open a new Chrome, it can't connect to an existing, running instance. Is it true?

Indeed! You'll have to close Chrome and then run your Playwright script for it to open a new one.

**Comment by iamandras** [comment]
> Indeed! You'll have to close Chrome and then run your Playwright script for it to open a new one.

Ok, got it :)

But using a user data dir on mac like below should work, right? Because the Windows equivalent works.
```
/Users/andras/Library/Application Support/Google/Chrome/
```
